app/Schema/Postgres.php

```diff
@@ -6,7 +6,12 @@
 use Kanboard\Core\Security\Token;
 use Kanboard\Core\Security\Role;
 
-const VERSION = 97;
+const VERSION = 98;
+
+function version_98(PDO $pdo)
+{
+    $pdo->exec('ALTER TABLE "comments" ADD COLUMN date_modification INT');
```

Comment:
You should use a BIGINT for Postgres as well, in this way we are ready for https://en.wikipedia.org/wiki/Year_2038_problem